src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPDocumentMappingProvider.cs

```diff
@@ -21,23 +21,23 @@ internal class DefaultLSPDocumentMappingProvider : LSPDocumentMappingProvider
         private static readonly TextEdit[] EmptyEdits = Array.Empty<TextEdit>();
 
         private readonly LSPRequestInvoker _requestInvoker;
-        private readonly LSPDocumentManager _documentManager;
+        private readonly Lazy<LSPDocumentManager> _lazyDocumentManager;
```

Comment:
Why did we wanna make this Lazy?

Reply:
https://github.com/dotnet/aspnetcore-tooling/pull/2041#discussion_r441098112